packages/editor/src/browser/editor-contribution.ts

```diff
@@ -0,0 +1,51 @@
+/*
+* Copyright (C) 2017 TypeFox and others.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+import { EditorManager } from './editor-manager';
+import { injectable, inject } from "inversify";
+import { StatusbarAlignment, Statusbar } from '@theia/core/lib/browser/statusbar/statusbar';
+import { FileIconProvider } from '@theia/filesystem/lib/browser/icons/file-icons';
+import { Position } from 'vscode-languageserver-types';
+
+@injectable()
+export class EditorContribution {
+
+    constructor(
+        @inject(Statusbar) protected readonly statusbar: Statusbar,
+        @inject(EditorManager) protected readonly editorManager: EditorManager,
+        @inject(FileIconProvider) protected readonly iconProvider: FileIconProvider
+    ) {
+        this.addStatusbarWidgets();
+    }
+
+    protected async addStatusbarWidgets() {
+
+        this.editorManager.onCurrentEditorChanged(async e => {
+            if (e) {
+                const langId = e.editor.document.languageId;
+                this.statusbar.setElement('editor-status-language', {
+                    text: langId.charAt(0).toUpperCase() + langId.substring(1),
```

Comment:
I think we should:
- add `Languages.languages`:
https://github.com/theia-ide/theia/blob/a96bb6105a579e211037cf8edc6713f628cae70e/packages/languages/src/common/languageclient-services.ts#L19-L21
- implement it in `MonacoLanguages` with `monaco.languages`:
https://github.com/theia-ide/theia/blob/a96bb6105a579e211037cf8edc6713f628cae70e/packages/monaco/src/browser/monaco-languages.ts#L21
- and use it here to get the language name